test/css/css_var_fallback_test.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require "components/test_helper"
+
+class Primer::CssVariableTest < Minitest::Test
+  def test_css_rules_all_contain_fallbacks
+    # 1. Find compiled CSS file
+    css_file = Rails.root.join(*%w(.. app assets styles primer_view_components.css)).read
```

Comment:
Rubocop wants this to be:

```suggestion
    css_file = Rails.root.join(*%w[.. app assets styles primer_view_components.css]).read
```